yb-voyager/cmd/importData.go

```diff
@@ -393,6 +404,13 @@ func splitFilesForTable(state *ImportDataState, filePath string, t string,
 			// handling possible case: last dataline(i.e. EOF) but no newline char at the end
 			numLinesTaken += 1
 		}
+		if line != "" && isDebeziumExport {
+			table := batchWriter.tableName
+			line, err = valueConverter.ConvertRow(table, TableToColumnNames[table], line)
```

Comment:
We don't need the `isDebeziumExport` flag, the `TableToColumnNames` map, and the `prepareTableToColumns()` function.

Just take the column names from the `ImportBatchArgs`. Also, it is okay to call the `ConvertRow()` for non-dbzm exports as well--because in that case it will be a no-op converter.